cmd/database.go

```diff
@@ -97,7 +110,16 @@ func savePgsql(jsonInfo string) {
 		exitWithError(err)
 	}
 
-	db, err := gorm.Open(postgres.Open(PsqlConnInfo.toString()), &gorm.Config{})
+	if PsqlConnInfo.Schema == "" {
```

Comment:
```suggestion
        var db *gorm.DB
	if PsqlConnInfo.Schema == "" {
```

How about moving the global variable `db` to here?

Reply:
Done